lib/utils.js

```diff
@@ -23,20 +24,24 @@ module.exports = {
     };
   },
 
-  walkSync(dir, filelist) {
-    var files = fs.readdirSync(dir);
-    filelist = filelist || [];
-    files.forEach((file) => {
-      if (fs.statSync(path.join(dir, file)).isDirectory()) {
-        filelist = this.walkSync(path.join(dir, file), filelist);
-      }
-      else {
-        filelist.push(path.join(path.basename(dir), file));
-      }
-    });
-    return filelist;
+  walk: function walk(dir) {
```

Comment:
Expanded this from shorthand form so we can safely call it recursively without getting reference from `this`. That was risky business from the start that would fail if someone destructures `walk(Sync)` at import site:
```js
const { walk } = require('./utils'); // this binding goes off...
```